src/m365/spo/commands/page/page-control-list.ts

```diff
@@ -31,39 +32,44 @@ class SpoPageControlListCommand extends SpoCommand {
   }
 
   public commandAction(logger: Logger, args: CommandArgs, cb: (err?: any) => void): void {
-    Page
-      .getPage(args.options.name, args.options.webUrl, logger, this.debug, this.verbose)
-      .then((clientSidePage: ClientSidePage): void => {
-        let controls: ClientSidePart[] = [];
-        clientSidePage.sections.forEach(s => {
-          s.columns.forEach(c => {
-            controls = controls.concat(c.controls);
-          });
-        });
-        // remove the column property to be able to serialize the array to JSON
-        controls.forEach(c => delete c.column);
+    let pageName: string = args.options.name;
+    if (args.options.name.indexOf('.aspx') < 0) {
+      pageName += '.aspx';
+    }
 
-        // remove the dynamicDataValues and dynamicDataPaths properties if they are null
-        controls.forEach(c => {
-          if (!c.dynamicDataPaths) {
-            delete c.dynamicDataPaths
-          }
-          if (!c.dynamicDataValues) {
-            delete c.dynamicDataValues
-          }
-        });
+    const requestOptions: any = {
+      url: `${args.options.webUrl}/_api/SitePages/Pages/GetByUrl('sitepages/${encodeURIComponent(pageName)}')`,
+      headers: {
+        'content-type': 'application/json;charset=utf-8',
```

Comment:
We don't need to specify the `content-type` request as this is a GET request and we're not sending anything.